atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1366,20 +1365,6 @@ void WebContents::DidFailLoad(content::RenderFrameHost* render_frame_host,
       is_main_frame);
 }
 
-void WebContents::DidGetResourceResponseStart(
```

Comment:
@bridiver this change requires and alternative for browser-laptop. From https://bugs.chromium.org/p/chromium/issues/detail?id=783981, `WebContentsObserver::DidFinishNavigation` is suggested as an alternative.

Reply:
can you use the alternative to emit the same event `did-get-response-details` to browser-laptop?

Reply:
I have pushed a commit here: https://github.com/brave/muon/tree/did-get-response-details
It breaks on startup because `GetResponseHeaders()` returns `null` sometimes, any idea about what to do in those cases?

Reply:
https://github.com/brave/muon/pull/537/commits/4b1636c209b6bb27e7fb8b5d96bdd4fe2d98010a
and you also missed emitting headers